src/flows/CostCalculator/CostCalculatorFlow.tsx

```diff
@@ -44,6 +46,17 @@ export type CostCalculatorFlowProps = {
   version?: CostCalculatorVersion;
 };
 
+const getDefaultManagementFee = (
+  baseRates: Record<CurrencyKey, number>,
+  currency: CurrencyKey,
+  managementFees?: Record<CurrencyKey, number>,
+) => {
+  if (managementFees && managementFees[currency]) {
+    return managementFees[currency];
+  }
+  return baseRates[currency] / 100;
```

Comment:
looks like if baseRates[currency] is undefined (i.e we don't provide a default), you return `NaN`, which is falsey, but looks like you might want to be explicit and guard against it, instead of letting the division do the work